spec/services/login_adoption_service_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe LoginAdoptionService do
+  let(:service) { LoginAdoptionService.new(user) }
+
+  context 'organic modal experiment' do
+    describe 'a dslogon user' do
+      let(:user) { create(:user, :dslogon) }
+
+      it 'organic modal flag is off' do
+        Flipper.disable(:organic_dsl_conversion_experiment)
```

Comment:
Generally anything that's part of test setup should go in a `let` or `before` block. In this case, a `before` block would be the proper place to disable the Flipper feature